ocs_ci/ocs/resources/storage_client.py

```diff
@@ -0,0 +1,98 @@
+"""
+StorageClient related functionalities
+"""
+
+import logging
+import tempfile
+import time
+
+
+# from ocs_ci.framework import config
+from ocs_ci.ocs import constants, ocp
+from ocs_ci.utility.retry import retry
+from ocs_ci.ocs.utils import (
+    run_cmd,
+)
+from ocs_ci.utility import templating
+from ocs_ci.ocs.resources.storage_cluster import (
+    check_storage_client_status,
+)
+from ocs_ci.helpers.managed_services import verify_storageclient_storageclass_claims
+
+
+log = logging.getLogger(__name__)
+
+
+@retry(AssertionError, 12, 10, 1)
+def create_storage_client(
+    storage_provider_endpoint=None,
+    onboarding_token=None,
+    expected_storageclient_status="Connected",
+):
+    """
+    This method creates storage clients
+
+    Inputs:
+    storage_provider_endpoint (str): storage provider endpoint details.
+    onboarding_token (str): onboarding token
+    expected_storageclient_status (str): expected storageclient phase; default value is 'Connected'
+
+    """
+    ocp_obj = ocp.OCP()
+    # Pull storage-client yaml data
+    log.info("Pulling storageclient CR data from yaml")
+    storage_client_data = templating.load_yaml(constants.STORAGE_CLIENT_YAML)
+    resource_name = storage_client_data["metadata"]["name"]
+    print(f"the resource name: {resource_name}")
+
+    # Check storageclient is available or not
+    storage_client_obj = ocp.OCP(kind="storageclient")
+    is_available = storage_client_obj.is_exist(
+        resource_name=resource_name,
+    )
+
+    # Check storageclaims available or not
+    cmd = "oc get storageclassclaim"
+    storage_claims = run_cmd(cmd=cmd)
```

Comment:
Please use exec_cmd,
 I think you will always get a string in this variable, even if we don't have StorageClients it will return the headers.

Reply:
Removed in latest commit.